packages/web-react/src/components/Pagination/README.md

```diff
@@ -132,6 +144,18 @@ This component extends the `PaginationButtonLink` component.
 | -------------------- | -------- | ------- | -------- | ------------------------------- |
 | `accessibilityLabel` | `string` | `Next`  | ✕        | Accessibility label of the link |
 
+## UncontrolledPagination props
+
+| Name                         | Type                           | Default    | Required | Description                                                         |
+| ---------------------------- | ------------------------------ | ---------- | -------- | ------------------------------------------------------------------- |
+| `accessibilityLabel`         | `string`                       | ✕          | ✕        | Accessibility label of the page links                               |
+| `accessibilityLabelNext`     | `string`                       | `Next`     | ✕        | Accessibility label of the next link                                |
+| `accessibilityLabelPrevious` | `string`                       | `Previous` | ✕        | Accessibility label of the previous link                            |
+| `defaultPage`                | `number`                       | `1`        | ✕        | The number of the page selected as current page at the first render |
+| `onChange`                   | `(pageNumber: number) => void` | ✕          | ✕        | On page change callback                                             |
+| `totalPages`                 | `number`                       | -          | ✕        | Total count of pages                                                |
+| `visiblePages`               | `number`                       | `5`        | ✕        | Number of displayed pages between chevrons                          |
```

Comment:
```suggestion
| `visiblePages`               | `number`                       | `5`        | ✕        | Number of displayed pages                        |
```

There is only one chevron when you are on the edge. Maybe we can say `numbered pages` or we can keep it as i suggest here ^